Library/Homebrew/utils.rb

```diff
@@ -101,9 +101,18 @@ def odie(error)
 end
 
 def pretty_duration(s)
-  return "2 seconds" if s < 3 # avoids the plural problem ;)
-  return "#{s.to_i} seconds" if s < 120
-  "%.1f minutes" % (s/60)
+  s = s.to_i
+  res = ""
+
+  if s > 59
+    m = s / 60
+    s = s % 60
+    res = "%d minute%s" % [ m, plural(m) ]
```

Comment:
Is there a reason for this style decision? String formatting isn't mentioned in GitHub's style guide and IMHO interpolation is less readable in this case.